lib/DDG/Meta/Data.pm

```diff
@@ -24,12 +24,9 @@ my %ia_metadata;
 # Only build metadata once. Not in BUILD so we can call apply_keywords directly
 unless(%ia_metadata){
 
-    my $tmpdir = io->tmpdir;
-    unless($tmpdir){
-       die 'No system temp directory found';
-    }
+    my $tmpdir = $ENV{METADATA_TMP_DIR} || '/var/tmp';
```

Comment:
could we default to `/var/tmp/ddg-metadata/` as per @jkv's suggestion from the Asana desc?